src/components/Form/Form.js

```diff
@@ -898,18 +906,16 @@ class Form extends Component {
             }
             case "student": {
                 let studentList = [];
-
                 if (this.props.currentParent) {
-                    this.props.parents[this.props.currentParent.user.id] &&
-                        this.props.parents[this.props.currentParent.user.id].student_ids.forEach((studentID) => {
-                            if (this.props.students[studentID]) {
-                                let { user_id, name, email } = this.props.students[studentID];
-                                studentList.push({
-                                    value: user_id,
-                                    label: `${name} - ${email}`,
-                                });
-                            }
-                        });
+                    this.props.currentParent.student_list.forEach((studentID) => {
+                        if (this.props.students[studentID]) {
+                            let { user_id, name, email } = this.props.students[studentID];
+                            studentList.push({
```

Comment:
The `if` and `push` can be replaced with an appropriate `.filter()` chained to a `.map()`

Reply:
This can't because the if is checking to see if the student exists in redux. if it doesn't exist in redux, then don't do anything. with that said, i think this can be taken out because in a later commit in this pull I made it so that we're fetching all the parent's students.